README.md

```diff
@@ -26,10 +28,10 @@ val analyze = Analyze { parquetFile =>
 }
 
 val sendReport = SendReport { newReportFile =>
-  sendReport(newReportFile)
+  sendReportToDashboard(newReportFile)
```

Comment:
@dmitri-carpov Renamed the method called by this stage and added the stage to the execution plan below